blocks/editable/index.js

```diff
@@ -472,15 +473,23 @@ export default class Editable extends Component {
 					</div>
 				}
 				<TinyMCE
-					tagName={ tagName }
+					tagName={ Tagname }
 					getSettings={ this.getSettings }
 					onSetup={ this.onSetup }
 					style={ style }
 					defaultValue={ value }
 					isEmpty={ this.state.empty }
-					placeholder={ placeholder }
+					label={ placeholder }
 					key={ key }
 				/>
+				{ this.state.empty &&
+					<Tagname
+						className="blocks-editable__tinymce"
```

Comment:
Wondering if we should include a modifier class here so reading the stylesheet is more obvious the intention on the `.blocks-editable__tinymce + .blocks-editable__tinymce` styling.